test/Sentry.Tests/SerializationTests.verify.cs

```diff
@@ -1,3 +1,7 @@
+#if NET7_0_OR_GREATER
```

Comment:
Wouldn't net6 work here? That's what the following added code from above made me believe:

```
 <!-- Ensure at least version 6 of System.Text.Json so we have JsonSerializationContext available -->
  <ItemGroup Condition="!$(TargetFramework.StartsWith('net6')) and !$(TargetFramework.StartsWith('net7')) and !$(TargetFramework.StartsWith('net8'))">
    <PackageReference Include="System.Text.Json" Version="6.0.8" />
```